Meta/Lagom/CMakeLists.txt

```diff
@@ -49,17 +47,14 @@ file(GLOB LIBREGEX_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibRegex/
 file(GLOB LIBCORE_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibCore/*.cpp")
 file(GLOB LIBELF_SOURCES CONFIGURE_DEPENDS "../../Userland/Libraries/LibELF/*.cpp")
 # There's no way we can reliably make this cross platform
-list(REMOVE_ITEM LIBELF_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/../../Userland/Libraries/LibELF/DynamicLinker.cpp")
-
-
+list(REMOVE_ITEM LIBELF_SOURCES "../../Userland/Libraries/LibELF/DynamicLinker.cpp")
```

Comment:
This won't remove the entry, file(GLOB) prepends the current directory to the resolved path.

Reply:
Fixed, ty!

Reply:
Added `CMAKE_CURRENT_SOURCE_DIR` to the other `REMOVE_ITEM` thing instead, no idea how that ever worked - maybe it didn't...